03-25 13:20:47 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-164]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1393)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1371)
	at org.h2.jdbc.JdbcConnection.createStatement(JdbcConnection.java:191)
	at com.thebrain.common.b.y.l(iuf:127)
	at com.thebrain.common.b.y.l(iuf:116)
	at com.thebrain.common.b.y.l(iuf:239)
	at com.thebrain.personal.a.a.i.g(yld:492)
	at com.thebrain.c.e.v.B(zdg:379)
	at com.thebrain.personal.a.a.k.c(wqd:307)
	at com.thebrain.personal.view.wg.l(txc:4231)
	at com.thebrain.personal.view.wg.l(txc:4176)
	at com.thebrain.personal.view.wg.l(txc:4214)
	at com.thebrain.personal.view.wc.run(txc:2760)
	at java.lang.Thread.run(Unknown Source)
03-25 13:20:49 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-164]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1393)
	at org.h2.jdbc.JdbcStatement.checkClosed(JdbcStatement.java:927)
	at org.h2.jdbc.JdbcPreparedStatement.checkClosed(JdbcPreparedStatement.java:1509)
	at org.h2.jdbc.JdbcStatement.checkClosed(JdbcStatement.java:901)
	at org.h2.jdbc.JdbcPreparedStatement.setParameter(JdbcPreparedStatement.java:1266)
	at org.h2.jdbc.JdbcPreparedStatement.setInt(JdbcPreparedStatement.java:308)
	at com.thebrain.common.model.p.l(nwf:834)
	at com.thebrain.common.model.p.l(nwf:1727)
	at com.thebrain.personal.a.a.i.l(yld:615)
	at com.thebrain.c.e.v.l(zdg:685)
	at com.thebrain.c.e.v.D(zdg:507)
	at com.thebrain.c.e.v.l(zdg:810)
	at com.thebrain.c.e.v.l(zdg:1080)
	at com.thebrain.c.b.s.l(uxf:297)
	at com.thebrain.c.b.s.m(uxf:95)
	at com.thebrain.c.b.s.l(uxf:232)
	at com.thebrain.c.b.g.run(uxf:30)
	at java.awt.event.InvocationEvent.dispatch(Unknown Source)
	at java.awt.EventQueue.dispatchEventImpl(Unknown Source)
	at java.awt.EventQueue.access$200(Unknown Source)
	at java.awt.EventQueue$3.run(Unknown Source)
	at java.awt.EventQueue$3.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$1.doIntersectionPrivilege(Unknown Source)
	at java.awt.EventQueue.dispatchEvent(Unknown Source)
	at com.thebrain.personal.b.d.z.dispatchEvent(gxc:205)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(Unknown Source)
	at java.awt.EventDispatchThread.pumpEventsForFilter(Unknown Source)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(Unknown Source)
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.awt.EventDispatchThread.run(Unknown Source)
03-25 13:20:50 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-164]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1393)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1371)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:265)
	at com.thebrain.common.b.x.L(wof:66)
	at com.thebrain.common.b.x.<init>(wof:243)
	at com.thebrain.common.b.p.l(lnf:211)
	at com.thebrain.common.b.p.l(lnf:229)
	at com.thebrain.common.model.p.l(nwf:904)
	at com.thebrain.common.model.p.l(nwf:1727)
	at com.thebrain.personal.a.a.i.l(yld:615)
	at com.thebrain.c.e.v.e(zdg:1203)
	at com.thebrain.c.e.v.l(zdg:636)
	at com.thebrain.c.e.v.l(zdg:1080)
	at com.thebrain.c.b.s.l(uxf:297)
	at com.thebrain.c.b.s.m(uxf:95)
	at com.thebrain.c.b.s.l(uxf:232)
	at com.thebrain.c.b.g.run(uxf:30)
	at java.awt.event.InvocationEvent.dispatch(Unknown Source)
	at java.awt.EventQueue.dispatchEventImpl(Unknown Source)
	at java.awt.EventQueue.access$200(Unknown Source)
	at java.awt.EventQueue$3.run(Unknown Source)
	at java.awt.EventQueue$3.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$1.doIntersectionPrivilege(Unknown Source)
	at java.awt.EventQueue.dispatchEvent(Unknown Source)
	at com.thebrain.personal.b.d.z.dispatchEvent(gxc:205)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(Unknown Source)
	at java.awt.EventDispatchThread.pumpEventsForFilter(Unknown Source)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(Unknown Source)
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.awt.EventDispatchThread.run(Unknown Source)
03-25 13:20:50 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-164]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1393)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1371)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:265)
	at com.thebrain.common.b.x.L(wof:66)
	at com.thebrain.common.b.x.<init>(wof:243)
	at com.thebrain.common.b.p.l(lnf:211)
	at com.thebrain.common.b.p.l(lnf:229)
	at com.thebrain.common.model.p.l(nwf:904)
	at com.thebrain.common.model.p.l(nwf:1727)
	at com.thebrain.personal.a.a.i.l(yld:615)
	at com.thebrain.c.e.v.e(zdg:1203)
	at com.thebrain.c.e.v.l(zdg:636)
	at com.thebrain.c.e.v.l(zdg:1080)
	at com.thebrain.c.b.s.l(uxf:297)
	at com.thebrain.c.b.s.m(uxf:95)
	at com.thebrain.c.b.s.l(uxf:232)
	at com.thebrain.c.b.g.run(uxf:30)
	at java.awt.event.InvocationEvent.dispatch(Unknown Source)
	at java.awt.EventQueue.dispatchEventImpl(Unknown Source)
	at java.awt.EventQueue.access$200(Unknown Source)
	at java.awt.EventQueue$3.run(Unknown Source)
	at java.awt.EventQueue$3.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$1.doIntersectionPrivilege(Unknown Source)
	at java.awt.EventQueue.dispatchEvent(Unknown Source)
	at com.thebrain.personal.b.d.z.dispatchEvent(gxc:205)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(Unknown Source)
	at java.awt.EventDispatchThread.pumpEventsForFilter(Unknown Source)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(Unknown Source)
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.awt.EventDispatchThread.run(Unknown Source)
03-25 13:20:50 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-164]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1393)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1371)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:265)
	at com.thebrain.common.b.x.L(wof:66)
	at com.thebrain.common.b.x.<init>(wof:243)
	at com.thebrain.common.b.p.l(lnf:211)
	at com.thebrain.common.b.p.l(lnf:229)
	at com.thebrain.common.model.p.l(nwf:904)
	at com.thebrain.common.model.p.l(nwf:1727)
	at com.thebrain.personal.a.a.i.l(yld:615)
	at com.thebrain.c.e.v.e(zdg:1203)
	at com.thebrain.c.e.v.l(zdg:636)
	at com.thebrain.c.e.v.l(zdg:1080)
	at com.thebrain.c.b.s.l(uxf:297)
	at com.thebrain.c.b.s.m(uxf:95)
	at com.thebrain.c.b.s.l(uxf:232)
	at com.thebrain.c.b.g.run(uxf:30)
	at java.awt.event.InvocationEvent.dispatch(Unknown Source)
	at java.awt.EventQueue.dispatchEventImpl(Unknown Source)
	at java.awt.EventQueue.access$200(Unknown Source)
	at java.awt.EventQueue$3.run(Unknown Source)
	at java.awt.EventQueue$3.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$1.doIntersectionPrivilege(Unknown Source)
	at java.awt.EventQueue.dispatchEvent(Unknown Source)
	at com.thebrain.personal.b.d.z.dispatchEvent(gxc:205)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(Unknown Source)
	at java.awt.EventDispatchThread.pumpEventsForFilter(Unknown Source)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(Unknown Source)
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.awt.EventDispatchThread.run(Unknown Source)
03-25 13:20:50 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-164]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1393)
	at org.h2.jdbc.JdbcStatement.checkClosed(JdbcStatement.java:927)
	at org.h2.jdbc.JdbcPreparedStatement.checkClosed(JdbcPreparedStatement.java:1509)
	at org.h2.jdbc.JdbcStatement.checkClosed(JdbcStatement.java:901)
	at org.h2.jdbc.JdbcPreparedStatement.getParameterMetaData(JdbcPreparedStatement.java:1255)
	at com.thebrain.common.b.y.l(iuf:1229)
	at com.thebrain.common.b.y.k(iuf:808)
	at com.thebrain.common.b.y.E(iuf:729)
	at com.thebrain.common.b.y.m(iuf:632)
	at com.thebrain.common.model.p.E(nwf:1647)
	at com.thebrain.personal.a.a.i.D(yld:1691)
	at com.thebrain.personal.a.a.i.l(yld:711)
	at com.thebrain.personal.a.a.i.a(yld:642)
	at com.thebrain.c.e.v.e(zdg:1238)
	at com.thebrain.c.e.v.l(zdg:636)
	at com.thebrain.c.e.v.l(zdg:1080)
	at com.thebrain.c.b.s.l(uxf:297)
	at com.thebrain.c.b.s.m(uxf:95)
	at com.thebrain.c.b.s.l(uxf:232)
	at com.thebrain.c.b.g.run(uxf:30)
	at java.awt.event.InvocationEvent.dispatch(Unknown Source)
	at java.awt.EventQueue.dispatchEventImpl(Unknown Source)
	at java.awt.EventQueue.access$200(Unknown Source)
	at java.awt.EventQueue$3.run(Unknown Source)
	at java.awt.EventQueue$3.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$1.doIntersectionPrivilege(Unknown Source)
	at java.awt.EventQueue.dispatchEvent(Unknown Source)
	at com.thebrain.personal.b.d.z.dispatchEvent(gxc:205)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(Unknown Source)
	at java.awt.EventDispatchThread.pumpEventsForFilter(Unknown Source)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(Unknown Source)
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.awt.EventDispatchThread.run(Unknown Source)
03-25 13:20:50 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-164]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1393)
	at org.h2.jdbc.JdbcStatement.checkClosed(JdbcStatement.java:927)
	at org.h2.jdbc.JdbcPreparedStatement.checkClosed(JdbcPreparedStatement.java:1509)
	at org.h2.jdbc.JdbcStatement.checkClosed(JdbcStatement.java:901)
	at org.h2.jdbc.JdbcPreparedStatement.setParameter(JdbcPreparedStatement.java:1266)
	at org.h2.jdbc.JdbcPreparedStatement.setInt(JdbcPreparedStatement.java:308)
	at com.thebrain.common.b.y.l(iuf:925)
	at com.thebrain.common.model.p.E(nwf:1612)
	at com.thebrain.personal.a.a.i.D(yld:1691)
	at com.thebrain.personal.a.a.i.l(yld:711)
	at com.thebrain.personal.a.a.i.a(yld:642)
	at com.thebrain.c.e.v.e(zdg:1238)
	at com.thebrain.c.e.v.l(zdg:636)
	at com.thebrain.c.e.v.l(zdg:1080)
	at com.thebrain.c.b.s.l(uxf:297)
	at com.thebrain.c.b.s.m(uxf:95)
	at com.thebrain.c.b.s.l(uxf:232)
	at com.thebrain.c.b.g.run(uxf:30)
	at java.awt.event.InvocationEvent.dispatch(Unknown Source)
	at java.awt.EventQueue.dispatchEventImpl(Unknown Source)
	at java.awt.EventQueue.access$200(Unknown Source)
	at java.awt.EventQueue$3.run(Unknown Source)
	at java.awt.EventQueue$3.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$1.doIntersectionPrivilege(Unknown Source)
	at java.awt.EventQueue.dispatchEvent(Unknown Source)
	at com.thebrain.personal.b.d.z.dispatchEvent(gxc:205)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(Unknown Source)
	at java.awt.EventDispatchThread.pumpEventsForFilter(Unknown Source)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(Unknown Source)
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.awt.EventDispatchThread.run(Unknown Source)
03-25 13:20:50 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-164]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1393)
	at org.h2.jdbc.JdbcStatement.checkClosed(JdbcStatement.java:927)
	at org.h2.jdbc.JdbcPreparedStatement.checkClosed(JdbcPreparedStatement.java:1509)
	at org.h2.jdbc.JdbcStatement.checkClosed(JdbcStatement.java:901)
	at org.h2.jdbc.JdbcPreparedStatement.setParameter(JdbcPreparedStatement.java:1266)
	at org.h2.jdbc.JdbcPreparedStatement.setInt(JdbcPreparedStatement.java:308)
	at com.thebrain.common.b.y.l(iuf:925)
	at com.thebrain.common.model.p.E(nwf:1612)
	at com.thebrain.personal.a.a.i.D(yld:1691)
	at com.thebrain.personal.a.a.i.l(yld:711)
	at com.thebrain.personal.a.a.i.a(yld:642)
	at com.thebrain.c.e.v.e(zdg:1238)
	at com.thebrain.c.e.v.l(zdg:636)
	at com.thebrain.c.e.v.l(zdg:1080)
	at com.thebrain.c.b.s.l(uxf:297)
	at com.thebrain.c.b.s.m(uxf:95)
	at com.thebrain.c.b.s.l(uxf:232)
	at com.thebrain.c.b.g.run(uxf:30)
	at java.awt.event.InvocationEvent.dispatch(Unknown Source)
	at java.awt.EventQueue.dispatchEventImpl(Unknown Source)
	at java.awt.EventQueue.access$200(Unknown Source)
	at java.awt.EventQueue$3.run(Unknown Source)
	at java.awt.EventQueue$3.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$1.doIntersectionPrivilege(Unknown Source)
	at java.awt.EventQueue.dispatchEvent(Unknown Source)
	at com.thebrain.personal.b.d.z.dispatchEvent(gxc:205)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(Unknown Source)
	at java.awt.EventDispatchThread.pumpEventsForFilter(Unknown Source)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(Unknown Source)
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.awt.EventDispatchThread.run(Unknown Source)
03-25 13:20:50 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-164]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1393)
	at org.h2.jdbc.JdbcStatement.checkClosed(JdbcStatement.java:927)
	at org.h2.jdbc.JdbcPreparedStatement.checkClosed(JdbcPreparedStatement.java:1509)
	at org.h2.jdbc.JdbcStatement.checkClosed(JdbcStatement.java:901)
	at org.h2.jdbc.JdbcPreparedStatement.setParameter(JdbcPreparedStatement.java:1266)
	at org.h2.jdbc.JdbcPreparedStatement.setInt(JdbcPreparedStatement.java:308)
	at com.thebrain.common.b.y.l(iuf:925)
	at com.thebrain.common.model.p.E(nwf:1612)
	at com.thebrain.personal.a.a.i.D(yld:1691)
	at com.thebrain.personal.a.a.i.l(yld:711)
	at com.thebrain.personal.a.a.i.a(yld:642)
	at com.thebrain.c.e.v.e(zdg:1238)
	at com.thebrain.c.e.v.l(zdg:636)
	at com.thebrain.c.e.v.l(zdg:1080)
	at com.thebrain.c.b.s.l(uxf:297)
	at com.thebrain.c.b.s.m(uxf:95)
	at com.thebrain.c.b.s.l(uxf:232)
	at com.thebrain.c.b.g.run(uxf:30)
	at java.awt.event.InvocationEvent.dispatch(Unknown Source)
	at java.awt.EventQueue.dispatchEventImpl(Unknown Source)
	at java.awt.EventQueue.access$200(Unknown Source)
	at java.awt.EventQueue$3.run(Unknown Source)
	at java.awt.EventQueue$3.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$1.doIntersectionPrivilege(Unknown Source)
	at java.awt.EventQueue.dispatchEvent(Unknown Source)
	at com.thebrain.personal.b.d.z.dispatchEvent(gxc:205)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(Unknown Source)
	at java.awt.EventDispatchThread.pumpEventsForFilter(Unknown Source)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(Unknown Source)
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.awt.EventDispatchThread.run(Unknown Source)
03-25 13:20:50 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-164]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1393)
	at org.h2.jdbc.JdbcStatement.checkClosed(JdbcStatement.java:927)
	at org.h2.jdbc.JdbcPreparedStatement.checkClosed(JdbcPreparedStatement.java:1509)
	at org.h2.jdbc.JdbcStatement.checkClosed(JdbcStatement.java:901)
	at org.h2.jdbc.JdbcPreparedStatement.setParameter(JdbcPreparedStatement.java:1266)
	at org.h2.jdbc.JdbcPreparedStatement.setInt(JdbcPreparedStatement.java:308)
	at com.thebrain.common.b.y.l(iuf:925)
	at com.thebrain.common.model.p.E(nwf:1612)
	at com.thebrain.personal.a.a.i.D(yld:1691)
	at com.thebrain.personal.a.a.i.l(yld:711)
	at com.thebrain.personal.a.a.i.a(yld:642)
	at com.thebrain.c.e.v.e(zdg:1238)
	at com.thebrain.c.e.v.l(zdg:636)
	at com.thebrain.c.e.v.l(zdg:1080)
	at com.thebrain.c.b.s.l(uxf:297)
	at com.thebrain.c.b.s.m(uxf:95)
	at com.thebrain.c.b.s.l(uxf:232)
	at com.thebrain.c.b.g.run(uxf:30)
	at java.awt.event.InvocationEvent.dispatch(Unknown Source)
	at java.awt.EventQueue.dispatchEventImpl(Unknown Source)
	at java.awt.EventQueue.access$200(Unknown Source)
	at java.awt.EventQueue$3.run(Unknown Source)
	at java.awt.EventQueue$3.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$1.doIntersectionPrivilege(Unknown Source)
	at java.awt.EventQueue.dispatchEvent(Unknown Source)
	at com.thebrain.personal.b.d.z.dispatchEvent(gxc:205)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(Unknown Source)
	at java.awt.EventDispatchThread.pumpEventsForFilter(Unknown Source)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(Unknown Source)
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.awt.EventDispatchThread.run(Unknown Source)
03-25 13:20:50 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-164]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1393)
	at org.h2.jdbc.JdbcStatement.checkClosed(JdbcStatement.java:927)
	at org.h2.jdbc.JdbcPreparedStatement.checkClosed(JdbcPreparedStatement.java:1509)
	at org.h2.jdbc.JdbcStatement.checkClosed(JdbcStatement.java:901)
	at org.h2.jdbc.JdbcPreparedStatement.setParameter(JdbcPreparedStatement.java:1266)
	at org.h2.jdbc.JdbcPreparedStatement.setInt(JdbcPreparedStatement.java:308)
	at com.thebrain.common.b.y.l(iuf:925)
	at com.thebrain.common.model.p.E(nwf:1612)
	at com.thebrain.personal.a.a.i.D(yld:1691)
	at com.thebrain.personal.a.a.i.l(yld:711)
	at com.thebrain.personal.a.a.i.a(yld:642)
	at com.thebrain.c.e.v.e(zdg:1238)
	at com.thebrain.c.e.v.l(zdg:636)
	at com.thebrain.c.e.v.l(zdg:1080)
	at com.thebrain.c.b.s.l(uxf:297)
	at com.thebrain.c.b.s.m(uxf:95)
	at com.thebrain.c.b.s.l(uxf:232)
	at com.thebrain.c.b.g.run(uxf:30)
	at java.awt.event.InvocationEvent.dispatch(Unknown Source)
	at java.awt.EventQueue.dispatchEventImpl(Unknown Source)
	at java.awt.EventQueue.access$200(Unknown Source)
	at java.awt.EventQueue$3.run(Unknown Source)
	at java.awt.EventQueue$3.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$1.doIntersectionPrivilege(Unknown Source)
	at java.awt.EventQueue.dispatchEvent(Unknown Source)
	at com.thebrain.personal.b.d.z.dispatchEvent(gxc:205)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(Unknown Source)
	at java.awt.EventDispatchThread.pumpEventsForFilter(Unknown Source)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(Unknown Source)
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.awt.EventDispatchThread.run(Unknown Source)
